packages/opentelemetry-node-tracer/src/instrumentation/PluginLoader.ts

```diff
@@ -54,13 +59,15 @@ export class PluginLoader {
    * {@link Plugin} interface and export an instance named as 'plugin'. This
    * function will attach a hook to be called the first time the module is
    * loaded.
-   * @param modulesToPatch A list of plugins.
+   * @param pluginConfig an object to configure the plugin.
```

Comment:
Could you make this comment a bit more descriptive? E.g. an object whose keys are plugin names and whose boolean values indicate whether to enable the plugin

Reply:
Done, thanks.